frontend/src/component/common/Table/cells/FeatureOverviewCell/FeatureOverviewCell.tsx

```diff
@@ -191,14 +192,64 @@ const Tags: FC<{
     );
 };
 
-const PrimaryFeatureInfo: FC<{
+const StyledDependencyLink = styled(Link)({
+    display: 'block',
+});
+
+const DependencyPreview: FC<{ feature: string; project: string }> = ({
```

Comment:
this is loaded with 500ms delay and preload the feature hook so when you go to the feature it's already in a cache. Delay prevents extra requests when moving your mouse around the screen